modules/mri_violations/test/TestPlan.md

```diff
@@ -13,28 +13,31 @@
   with details of why the scan failed.
 3. The `Problem` column with the entry `Could not identify scan type` should 
    display a popup with details of  why the scan type could not be identified.
-4. Go back to the MRI violations menu filter page and resolve issues with each of the
+4. The `Image File` column should have a link to 
+   the correct violated scan in the Brainbrowser module. Check that this link 
+   works and takes you to the Brainbrowser module.
```

Comment:
this is merged but the only comment i would have would be spelling it `BrainBrowser`

Reply:
@christinerogers I can add this little change if I make another PR for the release :)